README.md

```diff
@@ -1,24 +1,216 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <img src="logo.PNG" alt="logo" width="auto"  height="auto" />
+  <br/>
+  </div>
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 [Budget App] <a name="about-project"></a>
 
-* Database creation
+**[Buget App]** is a classic example of a blog website. It is a functional website that show the list of Categories and associated operations to each categories.
 
-* Database initialization
+**[Video Description](https://drive.google.com/file/d/1R-GI7cmCG04wLd3bvoDH8fmlJsp5wSMx/view?usp=sharing)**
```

Comment:
- [ ] As the first reviewer was not able to open the video he was also not able to give feedback about it. So as mentioned in the project requirements you should highlight a piece of the code that you find interesting or are proud of making. Kindly record a new video that meets all the requirements 🙏🏾 

<img width="1026" alt="Screenshot 2023-06-04 at 00 11 08" src="https://github.com/Outkarm/Budget-app/assets/74501523/d84b4224-cc90-435b-9c34-3b5c46b1fb0c">